### ISSUE TITLE
cant use quoteCharacter option

### ISSUE THREAD

**Issue by joonkui95** [tssajo/Minify#49]
i need change my all " to ' but i cant set the value

**Comment by tssajo** [comment]
@joonkui95 Need more info please. What type of file are you trying to minify? CSS? Javascript? HTML?

**Comment by joonkui95** [comment]
I need minify HTML file and replaced " to '

**Comment by tssajo** [comment]
@joonkui95 Try this setting (see the end of the line for what you try to achieve) :

```
"html-minifier_options": "--collapse-boolean-attributes --collapse-whitespace --html5 --minify-css --minify-js --preserve-line-breaks --process-conditional-comments --remove-comments --remove-empty-attributes --remove-redundant-attributes --remove-script-type-attributes --remove-style-link-type-attributes --quote--character '",
```

If the above does not work, you can try it like this:

```
"html-minifier_options": "--collapse-boolean-attributes --collapse-whitespace --html5 --minify-css --minify-js --preserve-line-breaks --process-conditional-comments --remove-comments --remove-empty-attributes --remove-redundant-attributes --remove-script-type-attributes --remove-style-link-type-attributes --quote--character \'",
```
BTW, what platform are you on? Windows or *nix?

**Comment by joonkui95** [comment]
is on windows 10 and using sublime text 3 
1st setting try result
![image](https://cloud.githubusercontent.com/assets/14049597/22435199/1cc66ed4-e75a-11e6-8c95-4847675f4353.png)
2nd setting try result
![image](https://cloud.githubusercontent.com/assets/14049597/22435161/e91ddda6-e759-11e6-8ba0-8ebbc41d7f0b.png)

**Comment by tssajo** [comment]
@joonkui95 Please try it like this then:

```
"html-minifier_options": "--collapse-boolean-attributes --collapse-whitespace --html5 --minify-css --minify-js --preserve-line-breaks --process-conditional-comments --remove-comments --remove-empty-attributes --remove-redundant-attributes --remove-script-type-attributes --remove-style-link-type-attributes --quote--character \"'\"",
```

Please let me know if this worked or not. Thanks!

**Comment by joonkui95** [comment]
@tssajo 
here is the result
can run with no error and no replace " to '
![image](https://cloud.githubusercontent.com/assets/14049597/22453125/63120d8c-e7b7-11e6-8765-2f6bb635f27e.png)

**Comment by tssajo** [comment]
@joonkui95 Thanks for all the testing! This bug will be fixed in the next version of the `Minify` plugin which I'll try to release later today. The syntax you'll need in your settings file need to be like this:
```
"html-minifier_options": "--collapse-boolean-attributes --collapse-whitespace --html5 --minify-css --minify-js --preserve-line-breaks --process-conditional-comments --remove-comments --remove-empty-attributes --remove-redundant-attributes --remove-script-type-attributes --remove-style-link-type-attributes --quote-character '",
```

**Comment by tssajo** [comment]
@joonkui95 I just released version 1.2.7 of `Minify`. After you update, this issue should be fixed. Please re-open this issue if you still experience the problem after upgrading to version 1.2.7. Thanks!

**Comment by naveedahmed1** [comment]
I have below tag in my html file:

`<link rel="stylesheet" href="styles.56ec616e3d29e712c590.css" media="print" onload="this.media='all'">`

When I run `html-minifier -o dist/index.html dist/index.html --remove-comments --collapse-whitespace --minify-js --minify-css`

I generates:

`<link rel="stylesheet" href="styles.56ec616e3d29e712c590.css" media="print" onload='this.media="all"'>`

i.e. surrounds onload in single quote.

I want to keep this tag as it is therefore I tried
 
`--quote-character "`

But with this option it generates:


`<link rel="stylesheet" href="styles.56ec616e3d29e712c590.css" media="print" onload="this.media=&#34;all&#34;">`

Can anyone please help?
